system_files/desktop/shared/usr/share/applications/Bazzite Landing.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+NoDisplay=false
+Terminal=false
+Exec=xdg-open https://universal-blue.discourse.group/docs?topic=8
+Icon=/usr/share/ublue-os/bazzite/welcome.svg
+Name=Bazzite Landing
+Comment=Announcements, newsletters, and the forums
+Categories=Utility;
```

Comment:
```suggestion
Categories=Utility;

```

Reply:
We might be doing this another way, but thanks for adding this fix if we end up going this route :smile: